### PR TITLE
*: cut v0.7.0

### PR DESCRIPTION
**Pull request by mxinden**
I would like to cut the v0.7.0 release. This should be delayed till #847 is merged.

I have copied and extended the release notes of v0.7.0-rc.0. Let me know if this is not the usual way to do it.

Thanks in advance for any feedback.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## v0.7.0 / 2017-05-19
```
Comment (grobie) [inline CHANGELOG.md]: So apparently  there were discussions in the past about when to use the `v` prefix and when not and the decision was to only use it in ambiguous contexts like git tags, but not in changelogs or releases. For consistency with all other prometheus projects, let's remove it here.

Comment (grobie) [inline CHANGELOG.md]: The date needs to be updated.

Comment (mxinden) [inline CHANGELOG.md]: @grobie Thanks! Updated this and past releases in the `CHANGELOG.md` file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
 * [ENHANCEMENT] Add ability to specify `filter` string and `silenced` flag on `api/v1/alerts` GET endpoint
 * [ENHANCEMENT] Update `cache-control` to prevent caching for web assets in general.
+* [ENHANCEMENT] Serve web assets by alertmanager instead of external CDN (See #846)
+* [ENHANCEMENT] Elide secrets in alertmanager config (See #840)
+* [ENHANCEMENT] AMTool: Move config file to a more consistent location (See #843)
 * [BUGFIX] Enable builds for Solaris/Illumos
+* [BUGFIX] Load web assets based on url path (See #323)
 
-## v0.6.2 / 2017-05-09
+## 0.7.0-rc.0 / 2017-05-19
```
Comment (grobie) [inline CHANGELOG.md]: We usually remove release candidates from the changelog.

Comment (mxinden) [inline CHANGELOG.md]: Changed. Thanks.

**Comment by w0rm** [comment]
We've tried it with @stuartnelson3  and found a bug, when clicking on the alert
screen shot 2017-06-08 at 15 13 34
 labels

**Comment by mxinden** [comment]
@w0rm  I have pushed a fix in PR #856 
Does this fix the issue for you?

**Comment by w0rm** [comment]
@mxinden yeah, this is fixes the issue. Thanks!

**Pull request merged**